Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -26,7 +26,9 @@
 
 from Orange.data import Table, Domain, StringVariable, RowInstance
 from Orange.data.util import get_unique_names_duplicates
-from Orange.widgets import widget, gui, settings
+from Orange.widgets import widget, gui
+from Orange.widgets.settings import (
+    DomainContextHandler, ContextSetting, Setting)
```

Comment:
I changed this to avoid pylint's complaints about shadowing `settings` in `migrate_settings`.